Makefile

```diff
@@ -31,7 +31,11 @@ lint:
 		--enable=nilerr \
 		./...
 
-test:
+testdeps:
```

Comment:
we should align with what's  used in .github/workflows/build-go.yaml, then remove the install directlives from there. Also, please  use @latest for external tools